packages/tc-ui/src/primitives/boolean/server.jsx

```diff
@@ -52,13 +52,13 @@ const Checkbox = ({
 const EditBoolean = props => {
 	const {
 		propertyName,
-		value,
 		disabled,
 		isNested,
 		parentType,
 		parentCode,
 		onChange,
 	} = props;
+	const value = typeof props.value === 'boolean' ? props.value : '';
```

Comment:
I feel like this is a breaking change... are there any potential risks with this?

Reply:
It's a port of this https://github.com/Financial-Times/treecreeper/pull/308/files#diff-0a84d66ba70a9ea00b0d958dee53c7b90a9d3cda89c5d7352a6d94c17719a104L6

I've basically distributed that file around the primitive components